openstack/mysql/install.sls

```diff
@@ -226,33 +227,3 @@ my.cnf template:
     - enable: True
     - watch:
       - file: /etc/mysql/my.cnf
-
```

Comment:
I certainly ran into issue...of course now i don't remember what setting xenial as a hard bind to controller_ip.  Not really sure why you are mucking with that as a part of this but just as an fyi